plugins/theme_arc/oomox_plugin.py

```diff
@@ -154,23 +154,24 @@ class Plugin(OomoxThemePlugin):
     #     'ROUNDNESS',
     # ]
 
-    theme_model_gtk = []
-
-    theme_model_options = [
+    theme_model_gtk = [
         {
             'key': 'ARC_WIDGET_BORDER_COLOR',
             'fallback_function': lambda colors: mix_theme_colors(
                 colors['BTN_BG'], colors['BTN_FG'],
                 0.75
             ),
             'type': 'color',
-            'display_name': _('GTK3 Widget Border Color'),
+            'display_name': _('Border'),
         },
+    ]
+
+    theme_model_options = [
         {
             'key': 'ARC_TRANSPARENCY',
             'type': 'bool',
             'fallback_value': True,
-            'display_name': _('Enable GTK3 Theme Transparency'),
+            'display_name': _('Enable Theme Transparency'),
```

Comment:
did they modified Arch theme what now it supports setting border colors and transparency outside gtk3?

if not renaming those could be misleading since people will be creating tickets why the setting didn't applied to their gtk2 or xfce theme

Reply:
This transparency setting is available on gnome-shell as well. For example:
- https://github.com/NicoHood/arc-theme/blob/master/common/gtk-3.0/_colors.scss.thpl#L58
- https://github.com/NicoHood/arc-theme/blob/master/common/gnome-shell/3.30/sass/_common.scss#L1644

Reply:
Regarding the border color: At least [`$button_border`](https://github.com/NicoHood/arc-theme/blob/master/common/gtk-3.0/_colors.scss.thpl#L85) is used for [gnome-shell theme](https://github.com/NicoHood/arc-theme/blob/master/common/gnome-shell/3.30/sass/_common.scss#L142).

P.S.I didn't carefully check, and I thought `ARC_WIDGET_BORDER_COLOR` is also used for  [`$borders_color`](https://github.com/NicoHood/arc-theme/blob/master/common/gtk-3.0/_colors.scss.thpl#L19), but it was wrong... Is that intentional?

Reply:
it should be used for both entry and button borders, so you found a bug in theme template :-)

so this change can be accepted in this case

still i would like to discuss what could be a good way to report in theme options list what some options available/unavailable in some specific environements

Reply:
ah, i found the button border, it seems fine: https://github.com/NicoHood/arc-theme/blob/master/common/gtk-3.0/_colors.scss.thpl#L85

Reply:
I'm wondering if it's really necessary to report that in detail, but if necessary, I think adding small dim label under the main label would be nice. (Maybe I'll add a mockup.)

Reply:
the same kind of label as it now used when you choose different style (Arc/Materia/Oomox)?

Reply:
> the same kind of label as it now used when you choose different style (Arc/Materia/Oomox)?

Yes it is! For example, in the "Border" row it should be like this:

```
----------------------------------------------
Border
GTK3, GNOME Shell, Cinnamon
----------------------------------------------
```

Reply:
and only show that hint if theme option available not for every variant, right?

Reply:
Yes. But personally I'm not a big fan of the idea:
> report in theme options list what some options available/unavailable in some specific desktop environments or toolkits

This could be just noisy rather than useful.

Reply:
mb tooltip?

Reply:
or small asterisk `*` with tooltip

Reply:
> small asterisk * with tooltip

Sounds very nice :+1:

Reply:
Or dim `?` icon may look more neat.

Reply:
information-symbol -- exactly what should fit the case

Reply:
asterisk could benefit as being less visual noisy, though it could be unclear

Reply:
also, should such tooltip happen on hover or on click

Reply:
> or warning icon with tooltip text not supported by GTK+2 theme

That would be the best :+1: 

> also, should such tooltip happen on hover or on click?

+1 for on hover :)